include/ua_plugin_securitypolicy.h

```diff
@@ -92,17 +112,17 @@ typedef struct {
      * @param channelContext the channelContext which contains information about
      *                       the keys needed to decrypt the message.
      * @param data the data to decrypt. The decryption is done in place. */
-    UA_StatusCode(*decrypt)(const UA_SecurityPolicy *securityPolicy,
-                            void *channelContext,
-                            UA_ByteString *data) UA_FUNC_ATTR_WARN_UNUSED_RESULT;
+    UA_StatusCode (*decrypt)(const UA_SecurityPolicy *securityPolicy,
```

Comment:
additional space not required

Reply:
This was inconsistent with the rest of the function pointer declarations and imho looks better with the space.

Reply:
Right, the look is better.
Still, can you move purely cosmetic changes into a different commit from the functional changes?

Reply:
A different commit or a different PR alltogether?

Reply:
Commit is enough.